templates/s3/AssetManagementPolicy.json.j2

```diff
@@ -0,0 +1,25 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucket"
+      ],
+      "Resource": [
+        "arn:aws:s3:::{{ k8s_s3_public_bucket }}",
+        "arn:aws:s3:::{{ k8s_s3_private_bucket }}",
```

Comment:
Not sure how strict these have to be but JSON doesn't allow trailing commas.

Reply:
done!